packages/grafana-ui/src/components/Table/TableNG/Cells/renderers.tsx

```diff
@@ -92,6 +92,14 @@ const CELL_RENDERERS: Record<TableCellOptions['type'], TableCellRenderer> = {
   [TableCellDisplayMode.Pill]: PILL_RENDERER,
 };
 
+const CELL_STYLES: Partial<Record<TableCellOptions['type'], TableCellStyles>> = {
+  [TableCellDisplayMode.ColorBackground]: getColorCellStyles,
+  [TableCellDisplayMode.ColorText]: getColorCellStyles,
+  [TableCellDisplayMode.DataLinks]: getDataLinksStyles,
+  [TableCellDisplayMode.JSONView]: getJsonCellStyles,
+  [TableCellDisplayMode.Pill]: getPillStyles,
+};
```

Comment:
same as the cell renderer, now if a new cell type is added which has custom styles, or if an existing cell type needs custom styles, you just set it up here.